release-notes.txt

```diff
@@ -1,3 +1,751 @@
+
+Hardware support:
+-----------------
+New support for the boards Adafruit-PyBadge and STM32 Nucleo-F439ZI as well as
+for the LPS22CH nano pressure sensor and MCP47xx DAC.
+
+
+Summary:
+-------
+328 pull requests, composed of 810 commits, have been merged since the
+last release, and 6 issues have been solved. 40 people contributed with
+code in 83 days. 1851 files have been touched with 50235 (+) insertions and
+18701 deletions (-).
+
+
+Notations used below
+====================
+
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+
+New features and changes
+========================
+
+Core
+----
++ core/sched: add sched_change_priority() (#17093)
++ core: add functionality to check queue state of another thread (#16174)
++ tests: add stack usage metrics (#17706)
+* core/rmutex: use atomic utils (#16919)
+
+System Libraries
+----------------
++ drivers/mtd_flashpage: add mtd_flashpage_t type (#17627)
++ makefiles/dependency_resolution: add outer loop for DEFAULT_MODULE
+  deps (#17632)
++ pkg/fatfs: fatfs_vfs: wire up format() (#14430)
++ pkg/mbedtls: initial pkg import to use entropy module (#15671)
++ sys/bitfield: add support for bit-wise bitfield operations (#17710)
++ sys/senml: add SenML modules (#16384)
++ sys/vfs: add vfs_mount_by_path() (#17661)
++ sys/xtimer: introduce `xtimer_is_set()` (#17630)
++ sys/ztimer: add ztimer64_xtimer_compat complete xtimer replace module (#17670)
++ vfs: Introduce reliable disk enumeration (#17660)
+* drivers/dose: make use of ringbuffer for RX (#17210)
+* examples/twr_aloha: refactor (#17406)
+* pkg/fatfs: enable exFAT support (#17798)
+* pkg/fatfs: implement statvfs() (#17634)
+* rust: test on stable (#17805)
+* sc_vfs: print file size (#17622)
+* sys/auto_init: custom auto-initialization sequence (simple) (#17794)
+* sys/event/timeout: remove alternative xtimer implementation (#17689)
+* sys/evtimer: remove deprecated evtimer_now_min (#17655)
+* sys/fido2: use ztimer instead of xtimer (#17753)
+* sys/posix/pthread: newlib compatibility (#17734)
+* sys/test_utils/print_stack_usage: work with small stacks (#17891)
+* sys/vfs: provide vfs_fsync() (#17621)
+* sys/xtimer: make xtimer_ztimer_compat default backend (#17721)
+* sys/ztimer doc: List prerequisites for successful use of ztimer_now (#17614)
+* sys/ztimer64/util.c: fix ztimer64_set_timeout_flag (#17561)
+* sys/ztimer: cleanup xtimer_compat.h for 32Bit only (#17690)
+* sys/ztimer: Name callback types (#17758)
+* sys: some simple xtimer->ztimer conversions (#17892)
+* sys/*timer: rework dependecies to ease backend switch, prefer
+  ztimer_xtimer_compat over xtimer_on_ztimer (#17811)
+* sys/syscalls: make gettimeofday() implementation optional (#17733)
+* sys/ztimer: auto-select ztimer_no_periph_rtt only for samd21 (#17786)
+* sys: sort out ztimer_xtimer_compat and ztimer64_xtimer_compat depes (#17732)
+* tests/pkg_mbedtls: improve entropy test case and doc (#17606)
```

Comment:
Should move to tests/ section

Reply:
unaddressed

Reply:
Thanks for the heads up - GitHub was hiding some of your comments and I wasn't aware of this "great" feature.